### DISCUSSION TITLE
How do i make song open a website

### DISCUSSION THREAD

**Discussion by Szymonkruk12** [FunkinCrew/Funkin#3734] [Q&A] [answered]
Please tell me how do i this in the base game because just like jeffy's endless aethos after beating aethos the game will crash and bring you to the sml wiki website and i wanna do in the base game please tell me how to do this?

**Comment by NotHyper-474** [comment, marked as answer]
You can use either `FlxG.openURL` or `WindowUtil.openURL` to do that

**Reply by Szymonkruk12** [reply]
how do i do this

**Reply by NotHyper-474** [reply]
I assume you're making a HScript mod (the ones you put in the mods folder), right?
